### PR TITLE
fix/btc-bitcore-change

### PR DESCRIPTION
**Pull request by alazarevski**
Change to bitcore and implement balance api calls
Updated tests

### CODE REVIEW

File: src/actions/bitcoinActions.js [hunk 1]
```diff
@@ -184,10 +187,19 @@ const setBitcoinAddressesAction = (addresses: string[]): SetBitcoinAddressesActi
 
 const updateBitcoinBalance = (
   address: string,
-  unspentTransactions: BitcoinUtxo[],
+  balance: Object,
```
Comment (jsidorenko) [inline src/actions/bitcoinActions.js]: can we type this?

File: src/models/Bitcoin.js [hunk 2]
```diff
@@ -50,3 +50,11 @@ export type BitcoinAddress = {
   address: string,
   updatedAt: number,
 };
+
+export type BitcoinBalance = {
+  [string]: {
```
Comment (jsidorenko) [inline src/models/Bitcoin.js]: pls name that key, like `[walletId: string]: {`

File: src/reducers/bitcoinReducer.js [hunk 3]
```diff
@@ -17,27 +17,34 @@
...
   data: {
     addresses: BitcoinAddress[],
     unspentTransactions: BitcoinUtxo[],
+    balances: BitcoinBalance,
   },
   creationFailed?: boolean,
 };
 
+export type UpdateUnspentTransactionsAction = {
+  type: "UPDATE_UNSPENT_TRANSACTIONS",
```
Comment (grilix) [inline src/reducers/bitcoinReducer.js]:
```suggestion
  type: 'UPDATE_UNSPENT_TRANSACTIONS',
```

File: src/services/insight.js [hunk 4]
```diff
@@ -44,11 +44,21 @@ export const sendRawTransactionToNode = async (rawtx: string) => {
...
 };
+
+export const getAddressBalanceFromNode = (address: string) => {
+  return fetch(`${BITCOIN_INSIGHT_URL}/address/${address}/balance`, {
+    method: 'GET',
+    headers: {
+      Accept: 'application/json',
+    },
+  })
+    .then(validateResponse('getAddressBalance'));
```
Comment (grilix) [inline src/services/insight.js]:
```suggestion
    .then(validateResponse('getAddressBalanceFromNode'));
```

**Pull request merged**